boards/x86/up_squared/up_squared_defconfig

```diff
@@ -0,0 +1,15 @@
+CONFIG_X86=y
+CONFIG_SOC_APOLLO_LAKE=y
+CONFIG_BOARD_UP_SQUARED=y
+CONFIG_CPU_APOLLO_LAKE=y
```

Comment:
what is the difference b/w CONFIG_CPU_APOLLO_LAKE & CONFIG_SOC_APOLLO_LAKE? i mean can we have any other configuration with CONFIG_CPU_APOLLO_LAKE other than CONFIG_SOC_APOLLO_LAKE?

Reply:
The SOC will select the CPU, so will remove the CPU_* here.